src/tests/Login.test.jsx

```diff
@@ -107,3 +107,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('input field', async () => {
```

Comment:
@khushishikhu . I thought we want to test if the input is not given (aka empty) a warning should be shown on the empty field as per what we've discussed [here](https://github.com/anitab-org/bridge-in-tech-web/issues/32#issuecomment-683351426). However the content shown here is for when the correct credentials are given the error will not be shown. We have that test already at the very top of the file ("allows the user to login successfully"). Can you please change it to checking if fields is/are empty?

Reply:
also, please be clear on the naming of the test. Naming it `input field` doesn't actually show what the purpose of the test. 😉